src/components/Item/Item.js

```diff
@@ -0,0 +1,41 @@
+import './style.scss';
+import { useState } from 'react';
+import {Link} from 'react-router-dom';
+import ItemDetail from '../ItemDetail/ItemDetail';
+
+export const Item = ({item}) => {
+    const {title, price, picture, free_shipping} = item;
+    const [click, setclick] = useState(false);
+
+    const handleClick = () => setclick(!click);
+
+    return (
+        <Link to='item' onClick={handleClick}>
```

Comment:
basta con definir la prop "to" para redirigir, Link implementa un <a /> cuyo "href" hace referencia a "to"